axiom/nr_segment_datastore.c

```diff
@@ -211,7 +211,6 @@ bool nr_segment_datastore_end(nr_segment_t** segment_ptr,
   scoped_metric
       = create_metrics(segment, duration, datastore_string, collection,
                        operation, &datastore, params->instance);
-
```

Comment:
Could you reformat this file in such a way that it doesn't show up in diff (`git checkout origin/dev -- axiom/nr_segment_datastore.c && git add `axiom/nr_segment_datastore.c && `git commit -m "restore original state" axiom/nr_segment_datastore.c` should do it)?

Reply:
fixed in [2140f54](https://github.com/newrelic/newrelic-php-agent/pull/958/commits/2140f543ceee3f31462b975ad05328d1eeb4653d)